pygenn/genn_groups.py

```diff
@@ -60,14 +81,14 @@ def pull_var_from_device(self, var_name):
         """
         self._model.pull_var_from_device(self.name, var_name)
 
-    def pull_extra_global_param_from_device(self, egp_name, size=1):
+    def pull_extra_global_param_from_device(self, egp_name, size=None):
         """Wrapper around GeNNModel.pull_extra_global_param_from_device
 
         Args:
         egp_name    --  string with the name of the variable
         size        --  number of entries in EGP array
         """
-        self._model.pull_extra_global_param_from_device(self.name, egp_name, size)
+        self._pull_extra_global_param_to_device(egp_name, size)
```

Comment:
I will admit I am not seeing fully to the bottom but is this really "pull ... to_device"? - seems against our normal naming conventions.

Reply:
good spot - definitely something bad going on there

Reply:
I think you just got the name wrong ... but as you matched it in the definition of the wrapper, it's not actually a bug. You can just refactor to the right name I think.